docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-august-2024.adoc

```diff
@@ -0,0 +1,1680 @@
+== Features Introduced in August 2024
+
+Learn what's new on Prisma® Cloud in August 2024.
+
+* <<announcement>>
+* <<new-features>>
+* <<api-ingestions>>
+* <<new-policies>>
+* <<policy-updates>>
+* <<new-compliance-benchmarks-and-updates>>
+* <<changes-in-existing-behavior>>
+* <<rest-api-updates>>
+//* <<deprecation-notices>>
+
+[#announcement]
+=== Announcement
+
+[cols="50%a,50%a"]
+|===
+|*Feature*
+|*Description*
+
+|*Prisma Cloud Data Security (PCDS) End of Sale*
+
+|Prisma Cloud Data Security Posture Management (DSPM) and Artificial Intelligence Security Posture Management (AISPM) is generally available (GA).
+
+*End of Sale Notice*
+
+With the GA of DSPM and AISPM, Prisma Cloud Data Security (PCDS) module is consigned to the End of Sale (EOS) status starting with the 24.8.1 release. Note the following important dates:
+
+* PCDS EOS will be effective on August 31, 2024.
+* Prisma Cloud tenants will no longer be able to subscribe to the PCDS module after September 1, 2024.
+* PCDS subscribed tenants can continue to use the PCDS module until its End of Life.
+* End of Life/End of Support will be effective on August 31, 2025 (one year post EOS).
+
+|===
+
+
+[#new-features]
+=== New Features
+
+[cols="30%a,70%a"]
+|===
+|*Feature*
+|*Description*
+
+|*AI Assisted Queries*
+
+tt:[Secure the Infrastructure]
+
+tt:[24.8.1] 
+
+//RLP-146585, - To Do, JJ to share the doc link
+
+|Enhancements to Prisma Cloud's query launcher allow you to use https://docs.prismacloud.io/en/enterprise-edition/content-collections/search-and-investigate/launch-your-query[AI assisted queries] to retrieve saved searches from your current tenant. Going beyond keyword matching, AI powered semantic searches provide ease of use when launching investigations in Prisma Cloud. For instance, typing “public facing” as a query, returns results with “reachable from untrusted internet sources” as well, because the saved search matches the meaning of the query "public facing". AI assisted search can be toggled on and off as needed. Use the feedback buttons as shown in the image below to provide your feedback on this feature. 
+
+image::ai-assisted-search.gif[]
+
+|*RQL for AWS Access Key Discovery*
+
+tt:[Secure the Infrastructure]
+
+tt:[24.8.1] 
+
+//RLP-146594
+
+|Prisma Cloud's RQL enhancements help you discover detailed information about Access Keys held by users, their activity, rotation, and usage. Available for AWS, the following RQL query helps you enforce zero trust best practices in your cloud environment:
+
+* Queries the number of days passed since the last usage of an access key
```

Comment:
Please move the query to the next line after the description. You can force a line break by adding (space + space) at the end of the description. For example,

Queries the number of days passed since the last usage of an access key + 

Update the first query to `source.cloud.accesskey.lastused.days (>, <, = )` and the second one to 

`source.cloud.accesskey.activekeys (<, >, =) (0, 1, 2)`

Reply:
Done